code/game/objects/structures/minimap.dm

```diff
@@ -172,4 +172,22 @@
 
 /obj/item/weapon/map_sovafghan/attack_self(mob/user)
 	update_icon()
-	examine(user)
\ No newline at end of file
+	examine(user)
+
+/obj/item/weapon/map_clash
+	desc = "The Bear clan king's battle plan."
+	name = "Battle plan"
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "portable_areamap"
+	throwforce = WEAPON_FORCE_HARMLESS
+	force = WEAPON_FORCE_HARMLESS
+	w_class = 1.0
+	flags = FALSE
+
+/obj/item/weapon/map_clash/examine(mob/user)
+	update_icon()
+	user << browse("<img src=clash_map.png></img>","window=popup;size=630x630")
```

Comment:
Make sure to include "clash_map.png" in the other file containing the maps.

Reply:
🤔